src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -1,49 +1,51 @@
+import { IPivotItemProps, IPivotProps, Pivot, PivotItem } from "office-ui-fabric-react";
 import * as React from "react";
-import * as AutoPilotUtils from "../../../Utils/AutoPilotUtils";
```

Comment:
what happened here? why are the orders messed up?

Reply:
I think it was a new formatting change, Not sure why it wasn't done when it was checked in and on files.